kytos/core/controller.py

```diff
@@ -123,7 +123,7 @@ def enable_logs(self):
     def start(self, restart=False):
         """Create pidfile and call start_controller method."""
         self.enable_logs()
-        if not restart:
+        if not restart and self.options.daemon is True:
```

Comment:
`self.options.daemon is True` or `self.options.daemon is False`?

Isn't the `pidfile` automatically created by `KytosDaemon` and we need to manually create it when we are not on the `daemon` mode?